src/index.html

```diff
@@ -11,8 +14,54 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="none"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page__body">
+    <div
+      class="product-card"
+      data-qa="card"
+    >
+      <img
+        class="product-card__image"
+        src="images/imac.jpeg"
+        alt="product photo"
+      />
+      <h2 class="product-card__product-title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+      <p class="product-card__product-code">Product code: 195434</p>
+      <div class="product-card__rating">
+        <div class="stars">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="reviews">Reviews: 5</p>
+      </div>
+      <div class="product-card__price">
+        <p class="product-card__price-name">Price:</p>
+        <p class="product-card__price-value">$2,199</p>
+      </div>
+      <bottom
+        data-qa="hover"
+        class="product-card__bottom-buy"
+      >
+        BUY
+      </bottom>
```

Comment:
The `<bottom>` tag is not a valid HTML element. It should be replaced with `<button>` to ensure proper functionality and semantic correctness.